roles/DCOS.bootstrap/tasks/main.yml

```diff
@@ -70,9 +70,41 @@
      map(attribute='dcos_installation') | map(attribute='version') | list |
      union([dcos['version_to_upgrade_from'] | default(dcos['version']) ]) | unique }}
 
-- name: Ceate install directory/genconf
+- name: Create install directory/genconf
   file: path={{ download_path }}/genconf state=directory mode=0755
 
+- name: Create bootstrap CA directory
+  file: path={{ download_path }}/genconf/ca state=directory mode=0700
+
+- name: "Generate PSK for exhibitor bootstrap"
+  shell: cat /dev/urandom | tr -dc 'a-z' | fold -w 16 | head -n1
+  register: random_psk_result
+  when: dcos.enterprise_dcos
+
+- name: "PSK contents"
+  shell: cat "{{ download_path }}/genconf/ca/psk"
+  register: psk_contents_result
+  when: dcos.enterprise_dcos
+
+- name: Write CA service PSK
+  copy:
+    dest: "{{ download_path }}/genconf/ca/psk"
+    mode: 0600
+    content: "{{ random_psk_result.stdout }}"
```

Comment:
this changes on every run

Reply:
this is intentional.

Reply:
so the PSK should change on every Ansible run by intention?